httplog/httplog_test.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2023 Sauce Labs Inc. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+package httplog
+
+import (
+	"testing"
+)
+
+func TestParseMode(t *testing.T) {
```

Comment:
Often it's easier to have two tests one for OK situations other for error validation.